app/javascript/components/miq-about-modal.jsx

```diff
@@ -75,37 +76,93 @@ class MiqAboutModal extends React.Component {
     const plugins = Object.keys(data.server_info.plugins).map((key) => {
       const val = data.server_info.plugins[key];
       return (
-        <AboutModal.VersionItem
-          label={val.display_name}
-          versionText={val.version}
-          key={key}
-        />
+        <p className="ModalItem" key={key}>
+          <strong>
+            {`${val.display_name}`}
+          </strong>
+          &emsp;
+          {val.version}
+        </p>
       );
     });
 
     return (
-      <AboutModal
-        dialogClassName={dialogClassName}
-        show={show}
-        onHide={hideModal}
-        productTitle={`${data.product_info.name_full} ${data.server_info.release}`}
-        logo={data.product_info.branding_info.logo}
-        altLogo={data.product_info.name_full}
-        trademarkText={data.product_info.copyright}
+      <Modal
+        aria-label="About Modal"
+        modalHeading={`${data.product_info.name_full} ${data.server_info.release}`}
+        open={show}
+        onRequestClose={() => { hideModal(); }}
+        passiveModal
+        className="about-modal"
       >
-        <AboutModal.Versions>
-          <AboutModal.VersionItem label={__('Version')} versionText={`${data.server_info.version}.${data.server_info.build}`} />
-          <AboutModal.VersionItem label={__('Server Name')} versionText={data.server_info.appliance || ''} />
-          <AboutModal.VersionItem label={__('Region')} versionText={data.region.region.toString()} />
-          <AboutModal.VersionItem label={__('Zone')} versionText={data.zone.name || ''} />
-          <AboutModal.VersionItem label={__('User Name')} versionText={data.identity.name} />
-          <AboutModal.VersionItem label={__('User Role')} versionText={data.identity.role} />
-          <AboutModal.VersionItem label={__('Browser')} versionText={browser.browser} />
-          <AboutModal.VersionItem label={__('Browser Version')} versionText={browser.version.toString()} />
-          <AboutModal.VersionItem label={__('Browser OS')} versionText={browser.OS} />
-          <br />
-          <a
-            style={{ color: 'white' }}
+        <ModalBody className="about-modal-body">
+          <p className="ModalItem">
```

Comment:
Is it possible to put  paragraph component to a separate component and pass label and value as props?

Reply:
@kavyanekkalapu is there any components you were thinking of using here? Not sure if there is any component we can use with a label and value prop here.